exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/impl/TestStringFunctions.java

```diff
@@ -1441,11 +1442,12 @@ public void testUpper() throws Exception {
             "upper('AbC aBc') as col_space,\n" +
             "upper('123ABC$!abc123.') as col_special,\n" +
             "upper('') as col_empty,\n" +
-            "upper(cast(null as varchar(10))) as col_null\n" +
+            "upper(cast(null as varchar(10))) as col_null,\n" +
+            "upper('привет, мир!') as col_national\n" +
```

Comment:
Also added here  
```+ "upper('ßßßßßßßßßß') as col_ss\n" + ```
 which should yield 
```"SSSSSSSSSSSSSSSSSSSS"``` 
as uppercase (even without Locale.GERMAN) . This was in part to check for a buffer overflow ....

Reply:
@oleg-zinovev there ignored unit test tow tests below (https://github.com/apache/drill/pull/1450/files#diff-01e4581e7fc41d7bc289363e5d7ab661L1471), just enable it and it should cover the use cases.

Reply:
@Ben-Zvi 
The test works, but I'm confused by the reason:
`ß` - 2 byte character, converting it to the upper case returns 2 single-byte `S` characters. Are there any other examples of such symbols?

Reply:
Awesome. Please remove changes in the test case since `testMultiByteEncoding` covers all that is needed.

Reply:
@Ben-Zvi 
I can not find any string function supporting UTF-16